### PR TITLE
chore: added contribution process guidelines in PR template as a comment

### PR DESCRIPTION
**Pull request by OndrejTuma**
[FX-2288]

### Description

Updated PR template with link to contribution guidelines
Plus fixed link to Component API principles

### How to test

Create a PR from alpha package

### Review

- [ ] Read [CONTRIBUTING.md](https://github.com/toptal/picasso/blob/master/CONTRIBUTING.md) and [Component API principles](https://github.com/toptal/picasso/blob/master/docs/api-principles.md)
- [ ] Annotate all `props` in component with documentation
- [ ] Create `examples` for component
- [ ] Ensure that deployed demo has expected results and good examples
- [ ] Ensure that tests pass by running `yarn test`
- [ ] Ensure that visuals tests pass by running `yarn test:visual`. If not - check the documentation [how to fix visual tests](https://github.com/toptal/picasso/blob/master/docs/contribution/visual-testing.md#fixing-broken-visual-tests-inside-a-pr)
- [ ] Ensure the changed/created components have not caused accessibility issues. [How to use accessibility plugin in storybook](https://github.com/toptal/picasso/blob/master/docs/contribution/accessibility.md).

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run whole pipeline
- `@toptal-bot run build` - Check build
- `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and
1. You have no extra requests.
2. You have optional requests.
   1. Add `nit: ` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because
1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:
1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>

[FX-2288]: https://toptal-core.atlassian.net/browse/FX-2288?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: .changeset/nervous-pianos-dress.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@toptal/picasso': patch
+---
+
+Added contribution guidelines in PR template as a comment
```
Comment (denieler) [inline .changeset/nervous-pianos-dress.md:5]: I think we don't need changesets for internal changes. Our users would not see any changes in Picasso product 🤷‍♂️

**Review by denieler — APPROVED** [review]

File: .changeset/nervous-pianos-dress.md (hunk 1 above)
Comment (OndrejTuma) [reply .changeset/nervous-pianos-dress.md:5]: agree, i will revert this change 👍 
thank you

**Comment by OndrejTuma** [comment]
@toptal-bot run package:alpha-release

**Comment by OndrejTuma** [comment]
@toptal-bot run package:alpha-release

**Comment by OndrejTuma** [comment]
@toptal-bot run package:alpha-release

**Comment by OndrejTuma** [comment]
@toptal-bot run package:alpha-release

**Comment by OndrejTuma** [comment]
test comment

**Review by teimurjan — APPROVED** [review]

**Review by dmaklygin1 — APPROVED** [review]

**Pull request merged**